packages/block-library/src/footnotes/index.php

```diff
@@ -142,7 +142,12 @@ static function( $post ) {
 
 			if ( $_gutenberg_revision_id ) {
 				$revision = get_post( $_gutenberg_revision_id );
```

Comment:
I know it's not part of this fix, but we could use `wp_get_post_revision` for its extra checks. Otherwise there's no difference. Might be more explicit too.

Mentioned in Core patch here: https://github.com/WordPress/wordpress-develop/pull/4892#pullrequestreview-1543133677